src/components/MarketDamageAQ.jl

```diff
@@ -0,0 +1,66 @@
+methane_on_crops = myloadcsv("data/Corp_yield/methane_crop_yield_value.csv")
+@defcomp MarketDamageAQ begin
+    country = Index()
+
+    # Incoming parameter: Global CH4 emissions from ch4emissions component
+    global_ch4_emissions = Parameter(index=[time], unit="Mtonne/year")
+
+    # Impact parameters: value per Mt methane emitted in year 1, year 2, ..., year 50
+    crop_yield_value_per_mton_ch4 = Parameter(index=[country, 50], unit="\$/Mtonne")
+
+    # Component variable to store total crop yield value
+    total_crop_yield_value = Variable(index=[time, country], unit="\$")
+
+    function run_timestep(p, v, d, t)
+        # Model Year
+        y_year = [2020, 2030, 2040, 2050, 2075, 2100, 2150, 2200, 2250, 2300]
```

Comment:
When this is all working, please change this to a parameter, so we aren't locked into this particular set of years.